src/algorithms/telemetry_decoder/gnuradio_blocks/galileo_telemetry_decoder_gs.cc

```diff
@@ -73,7 +74,14 @@ galileo_telemetry_decoder_gs::galileo_telemetry_decoder_gs(
         {
         case 1:  // INAV
             {
-                d_PRN_code_period_ms = static_cast<uint32_t>(GALILEO_E1_CODE_PERIOD_MS);
+                if (signal == '1')
+                    {
+                        d_PRN_code_period_ms = static_cast<uint32_t>(GALILEO_E1_CODE_PERIOD_MS);
+                    }
+                else if (signal == '7')
+                    {
+                        d_PRN_code_period_ms = static_cast<uint32_t>(GALILEO_E5B_CODE_PERIOD_MS * GALILEO_E5B_I_SECONDARY_CODE_LENGTH);
```

Comment:
Sir, I have done the parameter assignment for E5b signal in a way similar to Galileo E5a, since the two signals are similar.
Also, it was an intuition, the value for `GALILEO_E1_CODE_PERIOD_MS` is `4 ms`, and if we multiply code period with secondary code length, we get a value same as `4 ms`.

Reply:
ok, looks good to me.